Update the description of HA for Alertmanager in faq.md (#2297)
prometheus/docs: content/docs/introduction/faq.md

Yes, run identical Prometheus servers on two or more separate machines.
Identical alerts will be deduplicated by the [Alertmanager](https://github.com/prometheus/alertmanager).

> Alertmanager supports [high availability](https://github.com/prometheus/alertmanager#high-availability) by interconnecting multiple Alertmanager instances building an Alertmanager cluster. Instances of a cluster communicate on top of a gossip protocol managed via [Hashicorps Memberlist](https://github.com/hashicorp/memberlist) library. 

Reviewer: ```suggestion
Alertmanager supports [high availability](https://github.com/prometheus/alertmanager#high-availability) by interconnecting multiple Alertmanager instances to build an Alertmanager cluster. Instances of a cluster communicate using a gossip protocol managed via [HashiCorp's Memberlist](https://github.com/hashicorp/memberlist) library. 
```

> Alertmanager supports [high availability]((https://github.com/prometheus/alertmanager#high-availability)) by interconnecting multiple Alertmanager instances building an Alertmanager cluster. Instances of a cluster communicate on top of a gossip protocol managed via [Hashicorps Memberlist](https://github.com/hashicorp/memberlist) library. 

Reviewer 2: ```suggestion
Alertmanager supports [high availability](https://github.com/prometheus/alertmanager#high-availability) by interconnecting multiple Alertmanager instances building an Alertmanager cluster. Instances of a cluster communicate on top of a gossip protocol managed via [Hashicorps Memberlist](https://github.com/hashicorp/memberlist) library. 
```
